tests/integration/components/navbar-test.js

```diff
@@ -0,0 +1,26 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | navbar', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`<Navbar />`);
+
+    assert.dom(this.element).hasText('');
+
+    // Template block usage:
+    await render(hbs`
+      <Navbar>
+        template block text
+      </Navbar>
+    `);
+
+    assert.dom(this.element).hasText('template block text');
+  });
```

Comment:
Are we not writing tests?

Reply:
I am thinking of covering all the tests in separate PRs and will assign to other people as well who wants to learn testing.